.github/workflows/benchmarks-integration-tests.yml

```diff
@@ -0,0 +1,110 @@
+name: Benchmarks and integration tests
+
+on:
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      llvm_build_type:
+        description: "LLVM build type: debug | release"
+        required: true
+        default: "release"
+      compiler_tester_reference_branch:
+        description: "compiler-tester branch to use as a benchmark reference"
+        required: true
+        default: "main"
+      compiler_tester_candidate_branch:
```

Comment:
It is a tag, not branch, isn't it?
Let's rephrase it to: defaults to branch `main` if the tag doesn't exist.

Reply:
Fixed, please, take a look